core/controllers/skill_editor.py

```diff
@@ -56,6 +56,18 @@ def _require_valid_version(version_from_payload, skill_version):
 class SkillEditorPage(base.BaseHandler):
     """The editor page for a single skill."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
```

Comment:
Please add regex validation for this, check other files to see how is it done. Ditto below.

Reply:
Hi @vojtechjelinek 
Actually, there is a merge conflict between my branch and develop as shown by the oppia bot which I fixed and committed from the github itself.
Moreover, I searched many files throughout the codebase and find out this code to suit the purpose **(inside reader.py)**:

![Screenshot from 2021-11-19 03-30-30](https://user-images.githubusercontent.com/75239620/142504933-45175542-55f9-4416-b8f6-7bf2278323cc.png)

**lint check passed proof (after final commit):**

![Screenshot from 2021-11-19 03-29-19](https://user-images.githubusercontent.com/75239620/142504943-052189b6-bbd5-4ea7-8056-af7111c91f06.png)

Also, one thing that I have noticed is that lint checks are not run when committed directly from the github editor.

Correct me if I have done anything wrong here. 
Waiting for your further guidance.